samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -142,6 +153,12 @@ Vendor specific commands can be divided into different categories as follows:
   The two most significant bits are calculated by the DTM module.
   This is possible because the 6 least significant bits of all valid TX power values are unique.
   The TX power can be modified only when no Transmitter Test or Receiver Test is running.
+* If the Length field is set to ``3``(symbol ``NRF21540_ANTENNA_SELECT``), the Frequency field sets the nRF21540 FEM antenna.
+  The valid values are 0 - ANT1 enabled, ANT2 disabled or 1 - ANT1 disabled, ANT2 enabled.
```

Comment:
Proposing to list the valid values in a bulleted list. It would be clearer to see the values.

Reply:
Good idea. Done